src/config.ts

```diff
@@ -0,0 +1,62 @@
+import * as Vec2 from 'lib/vec2'
+
+export const config = {
+  scaling: window.devicePixelRatio,
+  originalFrameInterval: 20,
+  pixelsPerMeter: (diagonal: number) => 18 * diagonal / 2000,
+  duration: 60000,
+  wheelchair: false,
+  spriteSizeReduction: 0.33, // Should be equal to -resize in Makefile
+  zoom: {
+    min: 1,
+    max: 2,
+    convergenceDuration: 1000,
+  },
+  dropRate: {
+    smallTree: 130,
+    tallTree: 170,
+    jump: 20,
+    thickSnow: 20,
+    thickerSnow: 20,
+    rock: 100,
+    side: {
+      tallTree: 1000,
+    },
+    skierDirection: {
+      any: 50,
+    }
+  },
+  skier: {
+    activeSensoRatio: 1.5 / 3,
+    directionAmplitudeRatio: 0.6,
+    verticalPosRatio: 0.15,
```

Comment:
It would be nice to explain these, as I think they are quite crucial for the game's behaviour.

I have read the code at use site, but I am not sure I understand what they do.